packages/forms/src/UIForm/fieldsets/Array/Array.component.js

```diff
@@ -139,6 +139,7 @@ export default class ArrayWidget extends React.Component {
 				onRemove={this.onRemove}
 				renderItem={this.renderItem}
 				isCloseable={this.isCloseable()}
+				disabled={schema.disabled}
```

Comment:
The schema seems to be passed already. Why not use props.schema instead of passing props.schema.disabled in a new props ?

Reply:
yes, right. changed to `schema.disabled` instead.